eth/chains/base.py

```diff
@@ -125,6 +125,10 @@ class BaseChain(Configurable, ABC):
     def __init__(self) -> None:
         raise NotImplementedError("Chain classes must implement this method")
 
+    @property
+    def vm_configuration(self) -> Tuple[Tuple[int, Type['BaseVM']], ...]:
+        return self.vm_configuration
```

Comment:
This is going to cause infinite recursion.

Reply:
Also, I believe we already have a `get_vm_configuration` method that you can use.